pytorch_lightning/loggers/wandb.py

```diff
@@ -85,14 +86,15 @@ def __init__(
         self,
         name: Optional[str] = None,
         save_dir: Optional[str] = None,
-        offline: bool = False,
+        offline: Optional[bool] = False,
```

Comment:
I don't think the change here is necessary. `Optional[bool]` is equivalent to `Union[bool, None]`, and `offline` only accepts `bool`, right?

Reply:
It should work but actually I want to clean up the way we pass init parameters in a follow-up PR and suggest to just use kwargs.
The possible parameters has evolved and we could just refer to [wandb.init doc](https://docs.wandb.ai/ref/init)

Reply:
That makes sense.